examples/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "gtk-rs-examples"
 version = "0.0.1"
 authors = ["The Gtk-rs Project Developers"]
-autobins = false
```

Comment:
Why this change?

Reply:
Otherwise I would have to add a `[[bin]]` section which would clash with #267

Reply:
I merged #267 so maybe rebase? ;)

Reply:
Thanks! Done